1 entries in records
1 entries in records
❌ Error processing record address: (psycopg2.ProgrammingError) can't adapt type 'INTEGER'
[SQL: INSERT INTO schema_lookup (table_name, column_name, data_type, is_nullable, is_primary_key, source_db) VALUES (%(table_name__0)s, %(column_name__0)s, %(data_type__0)s, %(is_nullable__0)s, %(is_primary_key__0)s, %(source_db__0)s), (%(table_name__1)s,  ... 838 characters truncated ... , %(column_name__8)s, %(data_type__8)s, %(is_nullable__8)s, %(is_primary_key__8)s, %(source_db__8)s)]
[parameters: {'source_db__0': 'SQL Server', 'column_name__0': 'AddressID', 'data_type__0': INTEGER(), 'table_name__0': 'address', 'is_primary_key__0': True, 'is_nullable__0': False, 'source_db__1': 'SQL Server', 'column_name__1': 'AddressLine1', 'data_type__1': NVARCHAR(length=60, collation='SQL_Latin1_General_CP1_CI_AS'), 'table_name__1': 'address', 'is_primary_key__1': False, 'is_nullable__1': False, 'source_db__2': 'SQL Server', 'column_name__2': 'AddressLine2', 'data_type__2': NVARCHAR(length=60, collation='SQL_Latin1_General_CP1_CI_AS'), 'table_name__2': 'address', 'is_primary_key__2': False, 'is_nullable__2': True, 'source_db__3': 'SQL Server', 'column_name__3': 'City', 'data_type__3': NVARCHAR(length=30, collation='SQL_Latin1_General_CP1_CI_AS'), 'table_name__3': 'address', 'is_primary_key__3': False, 'is_nullable__3': False, 'source_db__4': 'SQL Server', 'column_name__4': 'StateProvinceID', 'data_type__4': INTEGER(), 'table_name__4': 'address', 'is_primary_key__4': False, 'is_nullable__4': False, 'source_db__5': 'SQL Server', 'column_name__5': 'PostalCode', 'data_type__5': NVARCHAR(length=15, collation='SQL_Latin1_General_CP1_CI_AS'), 'table_name__5': 'address', 'is_primary_key__5': False, 'is_nullable__5': False, 'source_db__6': 'SQL Server', 'column_name__6': 'SpatialLocation', 'data_type__6': NullType(), 'table_name__6': 'address', 'is_primary_key__6': False, 'is_nullable__6': True, 'source_db__7': 'SQL Server', 'column_name__7': 'rowguid', 'data_type__7': UNIQUEIDENTIFIER(), 'table_name__7': 'address', 'is_primary_key__7': False, 'is_nullable__7': False, 'source_db__8': 'SQL Server', 'column_name__8': 'ModifiedDate', 'data_type__8': DATETIME(), 'table_name__8': 'address', 'is_primary_key__8': False, 'is_nullable__8': False}]
(Background on this error at: https://sqlalche.me/e/20/f405)
❌ Error processing record address: (psycopg2.ProgrammingError) can't adapt type 'INTEGER'
[SQL: INSERT INTO schema_lookup (table_name, column_name, data_type, is_nullable, is_primary_key, source_db) VALUES (%(table_name__0)s, %(column_name__0)s, %(data_type__0)s, %(is_nullable__0)s, %(is_primary_key__0)s, %(source_db__0)s), (%(table_name__1)s,  ... 838 characters truncated ... , %(column_name__8)s, %(data_type__8)s, %(is_nullable__8)s, %(is_primary_key__8)s, %(source_db__8)s)]
[parameters: {'source_db__0': 'SQL Server', 'column_name__0': 'AddressID', 'data_type__0': INTEGER(), 'table_name__0': 'address', 'is_primary_key__0': True, 'is_nullable__0': False, 'source_db__1': 'SQL Server', 'column_name__1': 'AddressLine1', 'data_type__1': NVARCHAR(length=60, collation='SQL_Latin1_General_CP1_CI_AS'), 'table_name__1': 'address', 'is_primary_key__1': False, 'is_nullable__1': False, 'source_db__2': 'SQL Server', 'column_name__2': 'AddressLine2', 'data_type__2': NVARCHAR(length=60, collation='SQL_Latin1_General_CP1_CI_AS'), 'table_name__2': 'address', 'is_primary_key__2': False, 'is_nullable__2': True, 'source_db__3': 'SQL Server', 'column_name__3': 'City', 'data_type__3': NVARCHAR(length=30, collation='SQL_Latin1_General_CP1_CI_AS'), 'table_name__3': 'address', 'is_primary_key__3': False, 'is_nullable__3': False, 'source_db__4': 'SQL Server', 'column_name__4': 'StateProvinceID', 'data_type__4': INTEGER(), 'table_name__4': 'address', 'is_primary_key__4': False, 'is_nullable__4': False, 'source_db__5': 'SQL Server', 'column_name__5': 'PostalCode', 'data_type__5': NVARCHAR(length=15, collation='SQL_Latin1_General_CP1_CI_AS'), 'table_name__5': 'address', 'is_primary_key__5': False, 'is_nullable__5': False, 'source_db__6': 'SQL Server', 'column_name__6': 'SpatialLocation', 'data_type__6': NullType(), 'table_name__6': 'address', 'is_primary_key__6': False, 'is_nullable__6': True, 'source_db__7': 'SQL Server', 'column_name__7': 'rowguid', 'data_type__7': UNIQUEIDENTIFIER(), 'table_name__7': 'address', 'is_primary_key__7': False, 'is_nullable__7': False, 'source_db__8': 'SQL Server', 'column_name__8': 'ModifiedDate', 'data_type__8': DATETIME(), 'table_name__8': 'address', 'is_primary_key__8': False, 'is_nullable__8': False}]
(Background on this error at: https://sqlalche.me/e/20/f405)
Finished running source to staging in series
Finished running source to staging in series
Total time taken 0.55 seconds
Total time taken 0.55 seconds
